src/ILOPool.sol

```diff
@@ -148,31 +141,13 @@ contract ILOPool is
         }
 
         Position storage _position = _positions[tokenId];
-        require(raiseAmount <= saleInfo.maxCapPerUser - _position.raiseAmount, "UC");
+        require(raiseAmount <= saleInfo.maxRaisePerUser - _position.raiseAmount, "UC");
```

Comment:
So this doesn include the NFT transferring fix?

Reply:
see `_beforeTokenTransfer` below